plugin/fulltext/mysql-test/ngram/ngram_basic.inc

```diff
@@ -0,0 +1,25 @@
+--echo
+--eval CREATE TABLE t (lang TEXT, str TEXT, FULLTEXT(str) WITH PARSER ngram) ENGINE=$engine CHARSET $charset
+
+# Some character are not possible to represent in some encodings. Ignore them.
+--disable_warnings
+--disable_query_log
+# http://www.gerolf.org/doc/polyglot/polyglot.htm.
+INSERT IGNORE INTO t VALUES
+  ('Chinese', '他们为什么不说中文？'),
+  ('English', 'Why can''t they just speak English?'),
+  ('Japanese', 'なぜ、みんな日本語を話してくれないのか？'),
+  ('Korean', '세계의 모든 사람들이 한국어를 이해한다면 얼마나 좋을까');
```

Comment:
There is no question mark at the end of the Korean phrase.

Reply:
Added.

The source where I took these phrases from, had no question mark, but in other places on the Internet this phrase is cited with question mark present. Looks like in Korean, as in Japanese, question marks are optional.